manifests/config.pp

```diff
@@ -109,11 +118,11 @@
   -> file { 'consul config.json':
     ensure  => present,
     path    => "${consul::config_dir}/config.json",
-    owner   => $::consul::user_real,
-    group   => $::consul::group_real,
-    mode    => $::consul::config_mode,
-    content => consul::sorted_json($config_hash, $::consul::pretty_config, $::consul::pretty_config_indent),
-    require => File[$::consul::config_dir],
+    owner   => $consul::user_real,
```

Comment:
something seems to be very wrong here. No leading `::` for variables but for classes?